### DISCUSSION TITLE
RE7 DLSS black screen

### DISCUSSION THREAD

**Discussion by PesDmitryBarbos** [praydog/REFramework#870] [General Questions]
RE7 DLSS works only in flat mode, black screen in VR, both in OPENVR and OPENXR, if you turn off DLSS, the image is doubled

**Comment by JustMyma** [comment]
same

**Reply by vitor-hbr** [reply]
Where is the RE framework release that works with upscaling and VR? Just found the upscaler beta on the nexus for RE4 Remake, but that doesn't trigger VR for me on RE7. Also, which version of DLSS are you guys trying to use?
